packages/documentation-site/patternfly-docs/content/AI/conversation-design/chatbots.md

```diff
@@ -0,0 +1,64 @@
+---
+id: Chatbots
+section: AI
+subsection: Conversation design
+---
+
+import { Button, Card, CardBody, CardFooter, CardTitle, Divider, Gallery, GalleryItem } from '@patternfly/react-core';
+import ArrowRightIcon from '@patternfly/react-icons/dist/esm/icons/arrow-right-icon';
+
+## Writing for chatbots 
+
+When chatbots are design to meet the needs of your users, they can improve the overall UX of your product, because they are convenient, efficient, and persistent. 
+
+Chatbots are only as good as the writing that goes into them. The language they use must build trust and clearly establish the “rules” of the conversation.
+ 
+There 2 main types of chatbots:
+
+<Gallery hasGutter minWidths={{default:'400px'}}>
+<GalleryItem>
+<Card isFullHeight isLarge>
+<CardTitle>Rule-based chatbots</CardTitle>
+    <CardBody>Bots that are based on a set of predefined rules. These use a planned, guided dialog, and have a script that the user must stay on. The script is written by a human, including any microcopy, rules, prompts, and responses.</CardBody>
+    <CardFooter> <Button component="a" href="https://www.patternfly.org/ux-writing/chatbots" variant="link" icon={<ArrowRightIcon />} iconPosition="end"> View chatbot writing guidelines  </Button></CardFooter>
```

Comment:
Will this link exist?

Reply:
oops, I actually meant to delete this page!